### PR TITLE
integrate vote pool related components into eth backend

### PR DESCRIPTION
**Pull request by calmbeing**
### Description

add a description of your changes here...

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

File: eth/backend.go [hunk 1]
```diff
@@ -232,21 +244,62 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
...
-	// 	eth.voteSigner = signer
-	// }
-
-	// if voteManager, err := vote.NewVoteManager(eth.EventMux(), chainConfig, eth.blockchain, eth.voteJournal, eth.voteSigner); err == nil {
-	// 	eth.voteManager = voteManager
-	// }
-	// votePool := vote.NewVotePool(chainConfig, eth.blockchain, eth.voteManager, nil)
-
-	
+	dataDir := stack.Config().DataDir
```
Comment (realuncle) [inline eth/backend.go:247]: It is not proper to introduce such a long code in this function

File: eth/backend.go [hunk 2]
```diff
@@ -60,6 +66,12 @@ import (
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
+const (
```
Comment (realuncle) [inline eth/backend.go:69]: Please remove these

File: cmd/geth/blsaccountcmd.go [hunk 3]
```diff
@@ -205,6 +206,14 @@ func blsWalletCreate(ctx *cli.Context) error {
 		SkipMnemonicConfirm: true,
 	}
 
+	pdir := filepath.Join(cfg.Node.DataDir, BLSPassWordPath)
```
Comment (realuncle) [inline cmd/geth/blsaccountcmd.go:209]: Should not right.

Comment (calmbeing) [reply cmd/geth/blsaccountcmd.go]: Resolved

File: eth/backend.go (hunk 2 above)
Comment (calmbeing) [reply eth/backend.go]: Done

File: eth/backend.go (hunk 1 above)
Comment (calmbeing) [reply eth/backend.go]: Updated

**Review by realuncle — APPROVED** [review]

**Pull request merged**